File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonPdfs.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;
import java.util.List;

/**
 * Created by zawzaw on 04/01/18.
 */

public class GsonPdfs {

    @SerializedName("total")
    private int total;

    @SerializedName("data")
    private List<GsonPdf> pdfList;

    public int getTotal() {
        return total;
    }

    public List<GsonPdf> getPdfList() {
        return pdfList;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonPdfs.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 04/01/18.
 */

class GsonPdfs {

    @SerializedName("total")
    val total: Int = 0

    @SerializedName("data")
    val pdfList: List<GsonPdf>? = null

}
